### PR TITLE
Add cluster load metrics MERGEOK

### PR DESCRIPTION
**Pull request by bratseth**
or @gjoranv

### CODE REVIEW

File: metrics/src/main/java/ai/vespa/metrics/ConfigServerMetrics.java [hunk 1]
```diff
@@ -45,8 +44,13 @@ public enum ConfigServerMetrics implements VespaMetrics {
...
     ORCHESTRATOR_LOCK_ACQUIRE("orchestrator.lock.acquire", Unit.OPERATION, "Number of attempts to acquire zookeeper lock"),
     ORCHESTRATOR_LOCK_ACQUIRED("orchestrator.lock.acquired", Unit.OPERATION, "Number of times zookeeper lock was acquired"),
-    ORCHESTRATOR_LOCK_HOLD_LATENCY("orchestrator.lock.hold-latency", Unit.SECOND, "Time zookeeper lock was held before it was released");
+    ORCHESTRATOR_LOCK_HOLD_LATENCY("orchestrator.lock.hold-latency", Unit.SECOND, "Time zookeeper lock was held before it was released"),
 
+    // Node repository metrics
+    CLUSTER_COST("cluster.cost", Unit.DOLLAR_PER_HOUR, "The cost of the nodes allocated to a certain cluster, in $/hr"),
+    CLUSTER_LOAD_IDEAL_CPU("cluster.load.ideal.cpu", Unit.FRACTION, "The ideal cpu load of a certain cluster"),
+    CLUSTER_LOAD_IDEAL_MEMORY("cluster.load.ideal.memory", Unit.FRACTION, "The ideal memory load of a certain cluster"),
+    CLUSTER_LOAD_IDEAL_DISK("cluster.load.ideal.disk", Unit.FRACTION, "The ideal disk load of a certain cluster");
```
Comment (gjoranv) [inline metrics/src/main/java/ai/vespa/metrics/ConfigServerMetrics.java:53]: None of the other node-repo metrics have been defined here, but we should probably do it in another PR.

Comment (gjoranv) [inline metrics/src/main/java/ai/vespa/metrics/ConfigServerMetrics.java:53]: Should these metrics be added to the doc? None of the other node-repo metrics have been defined here, but that could just be an oversight.

File: config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java [hunk 2]
```diff
@@ -131,6 +131,11 @@ private static Set<Metric> getConfigServerMetrics() {
         addMetric(metrics, ConfigServerMetrics.ZK_CONNECTIONS.last());
         addMetric(metrics, ConfigServerMetrics.ZK_OUTSTANDING_REQUESTS.last());
 
+        addMetric(metrics, ConfigServerMetrics.CLUSTER_COST.last());
+        addMetric(metrics, ConfigServerMetrics.CLUSTER_LOAD_IDEAL_CPU.last());
+        addMetric(metrics, ConfigServerMetrics.CLUSTER_LOAD_IDEAL_MEMORY.last());
+        addMetric(metrics, ConfigServerMetrics.CLUSTER_LOAD_IDEAL_DISK.last());
```
Comment (gjoranv) [inline config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java:137]: The node-repo metrics are not reported via metrics-proxy, so there is no need to add them here.

File: metrics/src/main/java/ai/vespa/metrics/Unit.java [hunk 3]
```diff
@@ -12,6 +12,7 @@ public enum Unit {
     CONNECTION(BaseUnit.CONNECTION, "A link used for communication between a client and a server"),
     DOCUMENT(BaseUnit.DOCUMENT, "Vespa document, a collection of fields defined in a schema file"),
     DOCUMENTID(BaseUnit.DOCUMENTID, "A unique document identifier"),
+    DOLLAR_PER_HOUR(BaseUnit.MAGNITUDE, "$/hr"),
```
Comment (gjoranv) [inline metrics/src/main/java/ai/vespa/metrics/Unit.java:15]: If we need documentation for this metric, please add BaseUnits 'dollar' and 'hour' and declare the unit with per_unit hour.

**Review by gjoranv — CHANGES_REQUESTED** [review]

File: metrics/src/main/java/ai/vespa/metrics/ConfigServerMetrics.java (hunk 1 above)
Comment (bratseth) [reply metrics/src/main/java/ai/vespa/metrics/ConfigServerMetrics.java:53]: Yes, I don't know why but don't we want all metrics defined?

File: metrics/src/main/java/ai/vespa/metrics/Unit.java (hunk 3 above)
Comment (bratseth) [reply metrics/src/main/java/ai/vespa/metrics/Unit.java]: Done. It would be nice with some explanation of what baseunit is for.

**Comment by bratseth** [comment]
@gjoranv PTAL

File: metrics/src/main/java/ai/vespa/metrics/Unit.java [hunk 4]
```diff
@@ -81,13 +81,14 @@ private enum BaseUnit {
         CONNECTION("connection"),
         DOCUMENT("document"),
         DOCUMENTID("documentid"),
+        DOLLAR("dollar"),
         FAILURE("failure"),
         FILE("file"),
         FRACTION("fraction"),
         HIT("hit"),
+        HOUR("hout"),
```
Comment (gjoranv) [inline metrics/src/main/java/ai/vespa/metrics/Unit.java:89]: typo

File: metrics/src/main/java/ai/vespa/metrics/Unit.java [hunk 5]
```diff
@@ -81,13 +81,14 @@ private enum BaseUnit {
         CONNECTION("connection"),
         DOCUMENT("document"),
         DOCUMENTID("documentid"),
+        DOLLAR("dollar"),
```
Comment (gjoranv) [inline metrics/src/main/java/ai/vespa/metrics/Unit.java:84]: I'm wondering if "dollar" is ambiguous (AUSD, CAD), so I suggest to use 'USD' instead.

**Review by gjoranv — DISMISSED** [review]
Please fix typo and consider comment about USD

File: config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java (hunk 2 above)
Comment (gjoranv) [reply config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java:137]: These sets are only allow-lists for the metrics-proxy, so adding them is a NOP. But I guess they could be processed by the proxy some time in the future.

File: metrics/src/main/java/ai/vespa/metrics/Unit.java (hunk 5 above)
Comment (bratseth) [reply metrics/src/main/java/ai/vespa/metrics/Unit.java:84]: I think it's ok ...

File: metrics/src/main/java/ai/vespa/metrics/Unit.java (hunk 4 above)
Comment (bratseth) [reply metrics/src/main/java/ai/vespa/metrics/Unit.java]: done
